frame/composable-traits/src/dex.rs

```diff
@@ -36,13 +36,16 @@ pub trait Amm {
 	fn redeemable_assets_for_given_lp_tokens(
 		pool_id: Self::PoolId,
 		lp_amount: Self::Balance,
+		min_base_amount: Self::Balance,
+		min_quote_amount: Self::Balance,
```

Comment:
If this `/// Returns the amount of base & quote asset redeemable for given amount of lp token.`, why do we need to add min_base and min_quote parameters?

Reply:
Because we don't want a check for CannotRespectMinimuAmount in frontend. with those two parameters it is done in RPC.

Reply:
:+1: